cpu/kinetis_common/include/periph_cpu.h

```diff
@@ -166,6 +166,14 @@ typedef struct {
  */
 void gpio_init_port(gpio_t pin, uint32_t pcr);
 
+/**
+ * @brief Override uart_t
+ */
+#define HAVE_UART_T
+typedef uint8_t uart_t;
+#define UART_DEV(x)         (x)
+#define UART_UNDEF          0xff
```

Comment:
why overriding the default values here? At least the `UART_DEV(x)` macro is identical to the one in `uart.h`